### PR TITLE
add command GetContentByShortName

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/obis-observer/commands/downlink/GetContentByShortName.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    static readonly examples = examples;
+
+    static readonly hasParameters = true;
+
+    // data - only body (without header)
+    static fromBytes ( data: Uint8Array ) {
+        const buffer = new CommandBinaryBuffer(data);
+
+        return new GetContentByShortName({requestId: buffer.getUint8(), shortName: buffer.getUint8()});
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/GetContentByShortName.ts:69]:
```suggestion
        return new GetContentByShortName({requestId: buffer.getUint8(), shortName: buffer.getUint8()});
```
```suggestion
        return new GetContentByShortName({
            requestId: buffer.getUint8(),
            shortName: buffer.getUint8()
        });
```
?

File: src/obis-observer/commands/uplink/GetContentByShortNameResponseStr.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import Command, {TCommandExampleList} from '../../Command.js';
```
Comment (DarkPark) [inline src/obis-observer/commands/uplink/GetContentByShortNameResponseStr.ts:1]: there is a new `command size` field in docs

**Review by DarkPark — CHANGES_REQUESTED** [review]

File: src/obis-observer/commands/downlink/GetContentByShortName.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/downlink/GetContentByShortName.ts]: fixed

File: src/obis-observer/commands/uplink/GetContentByShortNameResponseStr.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/uplink/GetContentByShortNameResponseStr.ts:1]: added

**Pull request merged**